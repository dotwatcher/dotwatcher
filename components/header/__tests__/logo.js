import 'jest-styled-components';

import Logo from '../logo';
import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';

describe('<Logo />', () => {
	it('renders <Logo />', () => {
		const tree = renderer.create(<Logo>Page Title</Logo>).toJSON()
		expect(tree).toMatchSnapshot()
	});
});
