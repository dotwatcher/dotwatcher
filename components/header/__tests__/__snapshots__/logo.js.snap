// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> renders <Logo /> 1`] = `
.c0 {
  display: table-cell;
  margin: var(--spacing-none);
  font-weight: 600;
  line-height: 1;
}

<h1
  className="c0"
>
  Page Title
</h1>
`;
